backend/src/subject/models.py

```diff
@@ -7,8 +7,8 @@
 StudentSubject = Table(
     "student_subject",
     Base.metadata,
-    Column("uid", ForeignKey("website_user.uid"), primary_key=True),
-    Column("subject_id", ForeignKey("subject.id"), primary_key=True),
+    Column("uid", ForeignKey("website_user.uid", ondelete="CASCADE"), primary_key=True),
+    Column("subject_id", ForeignKey("subject.id", ondelete="CASCADE"), primary_key=True),
```

Comment:
Is deze cascade nodig als je de gebruiker overal verwijdert voor de gebruiker zelf te verwijderen?